interface.go

```diff
@@ -76,7 +76,7 @@ type Handler interface {
 	SendTrap(trap SnmpTrap) (result *SnmpPacket, err error)
 
 	// UnmarshalTrap unpacks the SNMP Trap.
-	UnmarshalTrap(trap []byte, useResponseSecurityParameters bool) (result *SnmpPacket)
+	UnmarshalTrap(trap []byte, useResponseSecurityParameters bool) (result *SnmpPacket, err error)
```

Comment:
Can we add a mention about this change in CHANGELOG.md

Reply:
Sure thing 👍  I tried to take inspiration from previous changes, let me know if I should update anything else in the Changelog